charts/grafana/ci/with-dashboard-values.yaml

```diff
@@ -0,0 +1,19 @@
+dashboards:
+  my-provider:
+    my-awesome-dashboard:
+      gnetId: 10000
+      revision: 1
+      datasource: Prometheus
+dashboardProviders:
+  dashboardproviders.yaml:
+    apiVersion: 1
+    providers:
+    - name: 'my-provider'
```

Comment:
If this section is used, should we rename my-provider to advantedge-provider?

Reply:
These are from the original helm chart which I did not modify.